### PR TITLE
Add fabric, corda, aurora to daml integration kit roadmap

### PR DESCRIPTION
**Pull request by dasormeter**
- Update DAML Integration Kit roadmap to include Hyperledger Fabric, Corda and Aurora
- Corrected links for Canton
- Small wording and formatting changes

### CODE REVIEW

File: docs/source/daml-integration-kit/index.rst [hunk 1]
```diff
@@ -96,35 +95,56 @@ support for running DAML.
...
 .. list-table::
-   :widths: 25 25 25 25
+   :widths: 20 20 20 20 20
    :header-rows: 1
 
    * - Ledger
      - Status
-     - Developer
-     - More
+     - Integration
```
Comment (bethaitman) [inline docs/source/daml-integration-kit/index.rst]: My guess is that this should be
```suggestion
     - Integration Developer
```

File: docs/source/daml-integration-kit/index.rst [hunk 2]
```diff
@@ -96,35 +95,56 @@ support for running DAML.
...
+     - Deployment
```
Comment (bethaitman) [inline docs/source/daml-integration-kit/index.rst]: And my understanding is that this should be
```suggestion
     - Deployment Platform
```

File: docs/source/daml-integration-kit/index.rst [hunk 3]
```diff
@@ -96,35 +95,56 @@ support for running DAML.
...
+     - In Development
+     - `Blockchain Technology Partners <https://blockchaintp.com/>`__
+     - `Sextant <https://blockchaintp.com/sextant/>`__     
+     - `press release :: June 2019
+       <https://hub.digitalasset.com/press-release/digital-asset-announces-daml-partner-integrations-with-hyperledger-fabric-r3-corda-and-amazon-aurora>`__
    * - :doc:`/tools/sandbox`
      - Stable
      - `Digital Asset <https://digitalasset.com/>`__
+     - TBD
      - `PostgreSQL <https://www.postgresql.org/>`__ backend coming soon
```
Comment (bethaitman) [inline docs/source/daml-integration-kit/index.rst]: This is actually done, right? We could link to https://docs.daml.com/tools/sandbox.html#running-with-persistence.

**Comment by bethaitman** [comment]
Oops, I didn't realise that that had the automerge tag on, I guess I shouldn't have approved.

If you could address the comments in a follow-up PR that'd be great!

**Comment by dasormeter** [comment]
@bethaitman -- yup will do right now in followup pr

**Comment by dasormeter** [comment]
follow-on PR addressing comments : https://github.com/digital-asset/daml/pull/1817

**Pull request merged**